scripts/compile-idl-js

```diff
@@ -5,8 +5,8 @@
 set -euo pipefail
 
 did_files_to_compile() {
```

Comment:
I guess this file will have conflicts with https://github.com/dfinity/nns-js/pull/125

Since they will all be in the same repo, it might make sense to change the script so that it does all the packagas at once. Or that you can choose a package.

Not for this PR, but something to remember.